scanpy_scripts/lib/_filter.py

```diff
@@ -195,8 +220,8 @@ def _get_filter_conditions(attributes, param, category, subset):
         pt_match = percent_top_pattern.match(cond_name)
         qv_match = qc_vars_pattern.match(cond_name)
         if found > 1:
-            raise click.ClickException(f'Ambiguous parameter "{name}" found in '
-                                       'both cell and gene table')
+            raise click.ClickException(f"Ambiguous parameter \"{name}\" found in "
+                                       "both cell and gene table")
```

Comment:
Yup. Galaxy currently says '{name}' when the error is triggered

Reply:
That must be caused by something else rather than single quotes. As I tested it in command line, it prints the expected attribute name when triggered.

Reply:
It's caused by a typo that omit the "f" preceding the quoted string, hence not forming a format string.

https://github.com/ebi-gene-expression-group/scanpy-scripts/blob/6313d73820cde8fc3a4fcba07d9a0fecc19ff555/scanpy_scripts/lib/_filter.py#L221

Reply:
Oh dammit. I did fix the missing 'f' in some places, but in others I could have sworn I fixed things by double-quoting. Must be going mad - apologies there.